_targets.R

```diff
@@ -18,6 +19,9 @@ list(
   tar_target(oldest_active_sites, find_oldest_sites(states, parameter)),
 
   # TODO: PULL SITE DATA HERE
+  tar_target(wi_data, get_site_data(oldest_active_sites[1], states[1], parameter)),
+  tar_target(mn_data, get_site_data(oldest_active_sites[2], states[2], parameter)),
+  tar_target(mi_data, get_site_data(oldest_active_sites[3], states[3], parameter)),
```

Comment:
It is curious to me that this pipeline works with this current state. Everything looks great except that you should just be passing all of `oldest_active_sites` to the `get_site_data()` function for now. The [first line of this function](https://github.com/whiteellie/ds-pipelines-targets-3/blob/main/1_fetch/src/get_site_data.R#L4) filters to the correct state! We dive into a more efficient way to do this as the course progresses, but for now, we will just use the function as-is.

I am just suprised it works because `oldest_active_sites[1]` just returns the first column from that data.frame. I would think it would break the pipeline!

Reply:
Upon further inspection, I don't see that your pipeline built `wi_data`, `mn_data`, and `mi_data` in your screenshot. Maybe you forgot to run `tar_make()` after saving your new `_targets.R` file?

Reply:
ahhhh. I think you're right. I didn't save.